plans/fix-1893-serve-v6-loopback.md

```
@@ -0,0 +1,276 @@
+
+### iter-1 の検証
+
+| ミューテーション | 結果 |
+|---|---|
+| bind を待たずに名乗る（= 指摘された挙動） | 2 red |
+| `v6LoopbackServed` を常に true にする | 1 red |
+| parent が disconnect していても送る | 1 red |
+
+**実機 A —— 順序**（既定 bind, port 34730）。ログ行番号で見て bind が banner より前:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)